### PR TITLE
Adding size 16 browser and operating system icons

### PR DESCRIPTION
**Pull request by amehta265**
The cypress design system is currently missing browser and operating system icons.

These icons will be used in IATR on the app side: [Link to Figma](https://www.figma.com/file/PAOozuCxtfetqP8fUVvg0y/branch/QkCp2F69X0qyLo7qnYQ67e/Cypress-App?node-id=21467%3A202325&t=b8ojjS1LkkNmQN21-0)

### CODE REVIEW

**Comment by elevatebart** [comment]
I think we should have them in here (it cost nothing and is useful everywhere).

I agree that we should remove all configuration from the icons since the color is part of the brand.

Thank you @amehta265

**Comment by elevatebart** [comment]
@mapsandapps Both?

File: icon-registry/icons/os-generic_x16.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+</g>
+<defs>
+<clipPath id="clip0_4669_8471">
+<rect width="16" height="16" fill="currentColor"/>
```
Comment (elevatebart) [inline icon-registry/icons/os-generic_x16.svg:7]: this is the only one where it makes sense to have the classes. But do you really need this con for your dev ?

**Review by elevatebart — COMMENTED** [review]
one class missing then I will add the changeset

File: icon-registry/icons/os-generic_x16.svg (hunk 1 above)
Comment (amehta265) [reply icon-registry/icons/os-generic_x16.svg:7]: Yes I need it when there are multiple groups for a test. I would display this icon and the number of operating systems in the groups e.g. `<os-generic/> 2 operating systems`

**Pull request merged**